packages/link/src/Link.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { createStyledComponent } from '@mineral-ui/component-utils';
+import linkTheme from './linkTheme';
+
+type Props = {
+  /** Content of the links. Can be anything, but see LinkBlock, LinkImage, and LinkTitle. */
+  children: MnrlReactNode,
+  /** URI tied to the Link */
+  href: string
+};
+
+const Root = createStyledComponent(
+  'a',
+  (props, baseTheme) => {
```

Comment:
Lines 29 – 31 should all be on one line. Not sure why Prettier didn't catch that.